docs/dev/setup.rst

```diff
@@ -654,3 +580,54 @@ HTTP. We need to extend the configuration in the file
 
 
 .. _Apollon Converter: https://github.com/ls1intum/Apollon_converter
+
+------------------------------------------------------------------------------------------------------------------------
+
+.. include:: setup/common-problems.rst
+
+------------------------------------------------------------------------------------------------------------------------
+
+.. TODO: ask Simon or someone if structure is alright
```

Comment:
@sleiss or/and @Mtze maybe you can answer me if the structure of setup/distributed.rst is correct (see the TODO)?
Asking as there were some changes where I wasn't sure about the hierarchy.

Reply:
got confirmation via Slack that it is correct